code/__HELPERS/game.dm

```diff
@@ -532,16 +530,17 @@
 		Mark.Grant(Nominee)
 		Nominee.mind.special_role = "Cult Master"
 		Nominee.update_action_buttons_icon()
+		Nominee.apply_status_effect(/datum/status_effect/cult_master)
 		SSticker.mode.set_antag_hud(Nominee,"cultmaster")
 		GLOB.cult_mastered = TRUE
 		for(var/datum/mind/B in SSticker.mode.cult)
-			if(isliving(B.current))
-				B.current << "<span class='cultlarge'>[Nominee] has the cult's support and is now their master. Follow their orders to the best of your ability!"
+			if(!B.current.incapacitated())

```

Comment:
not !, that's a negation; this proc returns TRUE if the target is incapacitated.